.github/workflows/build.yml

```diff
@@ -36,13 +37,17 @@ jobs:
         if: matrix.os == 'ubuntu-latest'
         uses: gradle/gradle-build-action@v2
         with:
+          gradle-home-cache-cleanup: true
+          cache-read-only: ${{ github.ref != 'refs/heads/master' && github.ref != 'refs/heads/version-2.0' }}
```

Comment:
Maybe also add mt-coroutines here?

Reply:
Will be dropped in 2.0 anyway, highly doubt that there will be any changes there.